src/components/user/user.js

```diff
@@ -0,0 +1,67 @@
+import React, {Component} from 'react';
+import {
+  Link,
+  BrowserRouter,
+  Route,
+  Switch,
+  withRouter,
+  browserHistory
+} from "react-router-dom";
+import {connect} from 'react-redux';
+let url =  'https://flatearth-api.herokuapp.com/api/v1/auth/secret/'
+
+
+class User extends Component{
+  
+  render(){
+  
+    
+const {token1} = this.props;
+var Li = ()=>{
+   console.log(`token1`, token1)
+  var obj = token1;
+      var f = [];
+      for (var prop in obj) {
```

Comment:
It can be written that way `Object.values(obj).map(prop => prop.name)`

using the for-in loop is bad in 95% or just "always" bad, you need to find another way to loop over elements